1.1/spec/17-Annex-B.adoc

```diff
@@ -6,76 +6,164 @@ This annex summarises all the functions defined in GeoSPARQL, providing descript
+
+1x xsd:anyURI | Geometry | ogc:geomLiteral | (Multi)Polygon(not DGGS),
+
+CellList(DGGS) | Yes | Yes
+| geof:geometryN | 1x ogc:geomLiteral | GeometryCollection(not DGGS) | xsd:double | xsd:double | Yes | No
+| geometryType | 1x ogc:geomLiteral | Geometry | xsd:string | | Yes | Yes |
+| getSRID | 1x ogc:geomLiteral | Geometry | xsd:anyURI | | Yes | Yes |
+| geof:intersection | 2x ogc:geomLiteral | 2x Geometry | ogc:geomLiteral | Polygon(not DGGS),
+
+CellList(DGGS) | Yes | Yes
+| is3D | 1x ogc:geomLiteral | Geometry | xsd:boolean | | Yes | Yes |
+| geof:isEmpty | 1x ogc:geomLiteral | Geometry | xsd:boolean | | Yes | Yes
+| isMeasured | 1x ogc:geomLiteral | Geometry | xsd:boolean | | Yes | Yes |
+| geof:isSimple | 1x ogc:geomLiteral | Geometry | xsd:boolean | | Yes | Yes
+| geof:length | 1x ogc:geomLiteral | Geometry | xsd:double | xsd:double | Yes | No
+| geof:numGeometries | 1x ogc:geomLiteral | Geometry(not DGGS) | xsd:double | xsd:double | Yes | No
+| projectTo | 1x ogc:geomLiteral | Geometry | geo:gmlLiteral | | Yes | Yes
+| spatialDimension | 1x ogc:geomLiteral | Geometry | xsd:integer | | Yes | Yes
+| geof:symDifference | 2x ogc:geomLiteral | 2x Geometry | (Multi)Polygon(not DGGS),
+
+CellList(DGGS) | Yes | Yes
+| geof:transform | 1x ogc:geomLiteral, 1x xsd:anyURI | Geometry | ogc:geomLiteral | Geometry | Yes | No
+| geof:union | 2x ogc:geomLiteral | 2x Geometry | ogc:geomLiteral | Polygon(not DGGS),
+
+CellList(DGGS) | Yes | Yes
+7+| **Serialization functions**
+| geof:asDGGS | 1x ogc:geomLiteral | Geometry | geo:dggsLiteral | | Yes | Yes
+| geof:asGeoJSON | 1x ogc:geomLiteral | Geometry | geo:geoJSONLiteral | | Yes | Yes
+| geof:asGML | 1x ogc:geomLiteral, 1x  xsd:string | Geometry | geo:gmlLiteral | | Yes | Yes
+| geof:asKML | 1x ogc:geomLiteral | Geometry | geo:kmlLiteral | | Yes | Yes
+| geof:asWKT | 1x ogc:geomLiteral | Geometry | geo:wktLiteral | | Yes | Yes
+7+| **Extent functions**
+| geof:getSRID | 1x ogc:geomLiteral | Geometry | xsd:anyURI | | Yes | Yes
+| geof:maxX | 1x ogc:geomLiteral | Geometry | xsd:double | | Yes | Yes
+| geof:maxY | 1x ogc:geomLiteral | Geometry | xsd:double | | Yes | Yes
+| geof:maxZ | 1x ogc:geomLiteral | Geometry | xsd:double | | Yes | Yes
+| geof:minX | 1x ogc:geomLiteral | Geometry | xsd:double | | Yes | Yes
+| geof:minY | 1x ogc:geomLiteral | Geometry | xsd:double | | Yes | Yes
+| geof:minZ | 1x ogc:geomLiteral | Geometry | xsd:double | | Yes | Yes
+7+| **Other Functions**
+| geof:relate | 2x ogc:geomLiteral | | xsd:string | xsd:string | Yes | Yes
+|===
+
+== B.2 GeoSPARQL to SFA Functions Mapping
+
+The following table indicates which GeoSPARQL non-topological query functions map to Simple Features Access (<<ISO19125-1>>) functions and in which GeoSPARQL version the functions are defined.
+
+Where the Simple Features Access function has the same name as the GeoSPARQL function, 'x' is recorded.
+
+[role="smalltext"]
+|===
+| GeoSPARQL Function | in 1.0 | in 1.1 | SFA
```

Comment:
It looks like some of the labels may be wrong here. For instance, isn't maxX a 1.1 function?

Reply:
I _think_ I've fixed the table!